sklearn/preprocessing/data.py

```diff
@@ -503,11 +511,10 @@ class StandardScaler(BaseEstimator, TransformerMixin):
 
     See also
     --------
-    :func:`sklearn.preprocessing.scale` to perform centering and
-    scaling without using the ``Transformer`` object oriented API
+    scale: Equivalent function without the object oriented API
 
-    :class:`sklearn.decomposition.RandomizedPCA` with `whiten=True`
-    to further remove the linear correlation across features.
+    :class:`sklearn.decomposition.PCA`
+        Further removes the linear correlation across features with 'whiten=True'.
```

Comment:
Should we mention both the `RandomizedPCA` with `PCA`? @GaelVaroquaux What's your opinion?